test/shared/src/main/scala/zio/test/Gen.scala

```diff
@@ -143,8 +143,26 @@ object Gen extends GenZIO with FunctionVariants {
   /**
    * A generator of strings. Shrinks towards the empty string.
    */
-  final val anyString: Gen[Random with Sized, String] =
-    Gen.string(Gen.anyChar)
+  final def anyString: Gen[Random with Sized, String] =
+    Gen.string(Gen.anyUnicodeChar)
+
+  /**
+   * A generator of Unicode characters. Shrinks toward '0'.
+   * Heavily inspired from:
+   * https://github.com/typelevel/scalacheck/blob/ab15a9fe012ca7c9feb48ee188e4d167df76d6ba/src/main/scala/org/scalacheck/Arbitrary.scala#L122-L134
+   */
+  final val anyUnicodeChar: Gen[Random, Char] = {
+    def unicodeRange(min: Int, max: Int): Gen[Random, Char] =
```

Comment:
This definitely works but in the future could be simplified to `Gen.char`.

Reply:
Oh, I didn't see that. Yeah that's much better and safer.